src/directions/main.ts

```diff
@@ -98,14 +99,19 @@ export default class MapLibreGlDirections extends MapLibreGlDirectionsEvented {
   }
 
   protected async fetchDirections(originalEvent: MapLibreGlDirectionsWaypointEvent) {
+    this.abortController?.abort();
```

Comment:
Please, leave a comment why we need this. I know, but future contributors will get confused.